docs/pages/getting-started/running-docker.rst

```diff
@@ -0,0 +1,106 @@
+.. _running-docker:
+
+Getting Started on Docker
+#########################
+
+If you already use Docker, the easiest way to get your Nuts Node up and running for development or production is
+using Docker. To use the latest `master` build use `nutsfoundation/nuts-node:master` (for production environments
+it's advisable to use a specific version).
+
+First determine the working directory for the Nuts node which will contain configuration and data. These which will be mounted into the Docker container.
+
+Node TLS Certificate
+********************
+
+To connect to an existing Nuts network you need a TLS certificate which authenticates your node. For the development network
+you can use the `nuts-network-development-ca` to directly issue a certificate for your node. The commands below clone
+the required Git repository, generate a private key and issues a certificate, and combines them into a single file:
+
+```
+git clone https://github.com/nuts-foundation/nuts-development-network-ca
+cd nuts-development-network-ca && ./issue-cert.sh localhost
+cat localhost.key localhost.pem > certificate-and-key.pem
+```
+
+Move `certificate-and-key.pem` to the working directory.
+
+.. note::
+
+    If you want peers to be able to connect to your node, replace `localhost` with the correct hostname.
+
+Note that the Git repository contains the Certificate Authority certificate (`ca.pem`) which will function as truststore.
+Copy this file as `truststore.pem` into the working directory.
+
+YAML Configuration File
+***********************
+
+Copy the YAML file below and save it as `nuts.yaml` in the working directory:
+
+```
+datadir: /opt/nuts
+network:
+  trustStoreFile: /opt/nuts/truststore.pem
+  certFile: /opt/nuts/certificate-and-key.pem
+  certKeyFile: /opt/nuts/certificate-and-key.pem
+```
+
+See :ref:`configuration <nuts-node-config>` for more information on what can be configured.
```

Comment:
Athough you mentioned the `bootstrapNodes` option above, I thing it would help adding a reference here. Perhaps just suggesting to ask for the latest node using the slack channel?

Reply:
We were discussing this in Slack (do we need a Nuts bootstrap node for development > if not, should we provide nodes of community-driven nodes? > if so, how?). So when we decided we'll document it.